packages/cli/src/modules/dynamic-credentials.ee/database/entities/dynamic-credential-entry.ts

```diff
@@ -0,0 +1,39 @@
+import { CredentialsEntity, WithTimestamps } from '@n8n/db';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from '@n8n/typeorm';
+
+import { DynamicCredentialResolver } from './credential-resolver';
+
+@Entity({
+	name: 'dynamic_credential_entry',
+})
+export class DynamicCredentialEntry extends WithTimestamps {
+	constructor() {
+		super();
+	}
+
+	@PrimaryColumn({
+		name: 'credential_id',
+	})
+	credentialId: string;
+
+	@PrimaryColumn({
+		name: 'subject_id',
+	})
+	subjectId: string;
+
+	@PrimaryColumn({
+		name: 'resolver_id',
+	})
+	resolverId: string;
+
+	@Column('text')
+	data: string;
+
+	@ManyToOne(() => CredentialsEntity)
```

Comment:
Does this work without setting the other way OnyToMany on the CredentialsEntity ?

Reply:
I think so, but I'll write a test to check 👍

Reply:
I added tests to filter and fetch the relations on the entity. Seems to work fine.